modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -1,19 +1,25 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
+// * post_installation_configuration/cluster-tasks.adoc
 
 [id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
 = Binding infrastructure node workloads using taints and tolerations
 
 If you have an infra node that has the `infra` and `worker` roles assigned, you must configure the node so that user workloads are not assigned to it.
 
+[IMPORTANT]
+====
+It is recommended to preserve the dual `infra,worker` label that is created for infra nodes and use taints and tolerations to manage nodes user workloads are scheduled on. If you remove the `worker` label from the node, you must create a custom pool to manage it; a node with a label other than `master` or `worker` is not recognized by the MCO without a custom pool. Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, and the `infra` label communicates to the cluster that it does not count toward the total number of subscriptions.
+====
+
 .Prerequisites
 
 * Configure additional `MachineSet` objects in your {product-title} cluster.
 
 .Procedure
 
-. Use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
+. If you have not configured a taint on your machine set, use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
```

Comment:
Should we show users how to validate that node's don't have taints?

Reply:
@sferich888 The branch behins this PR is a clone of Cody's branch, with my changes. 

I added a sub-step to look in the node spec to see if there is a toleration. PTAL 
https://github.com/openshift/openshift-docs/pull/33057/commits/6d888fb5f5fc4839c90a11110a1f2efe78463617